compiler/rustc_const_eval/src/transform/check_consts/ops.rs

```diff
@@ -580,16 +580,21 @@ impl<'tcx> NonConstOp<'tcx> for StaticAccess {
         if let hir::ConstContext::Static(_) = ccx.const_kind() {
             Status::Allowed
         } else {
-            Status::Forbidden
+            Status::Unstable(sym::const_refs_to_static)
         }
     }
 
     fn build_error(&self, ccx: &ConstCx<'_, 'tcx>, span: Span) -> DiagnosticBuilder<'tcx> {
-        ccx.dcx().create_err(errors::StaticAccessErr {
+        let mut err = feature_err(
+            &ccx.tcx.sess.parse_sess,
+            sym::const_refs_to_static,
             span,
-            kind: ccx.const_kind(),
-            teach: ccx.tcx.sess.teach(&error_code!(E0013)).then_some(()),
-        })
+            format!("referencing statics in {}s is unstable", ccx.const_kind(),),
+        );
+        err
+            .note("`static` and `const` variables can refer to other `const` variables. A `const` variable, however, cannot refer to a `static` variable.")
+            .help("To fix this, the value can be extracted to a `const` and then used.");
```

Comment:
```suggestion
            .note("`static` and `const` variables can refer to other `const` variables; a `const` variable, however, cannot refer to a `static` variable")
            .help("to fix this, the value can be extracted to a `const` and then used");
```

https://rustc-dev-guide.rust-lang.org/diagnostics.html?highlight=lowercase#diagnostic-output-style-guide (subitem 2)